CMakeLists.txt

```diff
@@ -99,3 +106,24 @@ endif()
 add_subdirectory(matrix_multiplication)
 add_subdirectory(fft)
 add_subdirectory(dependent_index_sets)
+
+set(FIGURES_DIRECTORY "${CMAKE_SOURCE_DIR}/figs")
+
+add_custom_command(
+  OUTPUT rajaMmult.cali
+  COMMAND find "${CMAKE_BINARY_DIR}/matrix_multiplication/*.cali" | tail -n 1 | xargs -I '{}' mv '{}' ${CMAKE_BINARY_DIR}/caliper_files/rajaMmult.cali
+)
+
+add_custom_command(
+  OUTPUT "${FIGURES_DIRECTORY}/rajaMatrixMultiplication.pdf"
+  COMMAND python "${CMAKE_SOURCE_DIR}/analyze_caliper.py" raja mmult 
```

Comment:
But now we can just pass some parameters to this script and have it generate the appropriate file.